Swift/Database.swift

```diff
@@ -591,4 +599,24 @@ extension CBLDatabase {
             }
         })
     }
+    
+    /// The same as `inBatch(() -> Void)`, but the closure can return a Bool.
+    /// If the closure returns `false`, the transaction will be aborted, and this function will return false.
+    internal func maybeBatch(block: () throws -> Bool) throws -> Bool {
```

Comment:
Can this and `internal func maybeBatch(using block: () throws -> Bool ) throws -> Bool {` be combined?

Reply:
From slack chat, I think I got it.